js/npm/arangojs/Database.hx

```diff
@@ -59,6 +59,8 @@ implements npm.Package.RequireNamespace<"arangojs", "^4.3.0">
 	// Queries
 	// aqlQuery macro
 
+        @:overload(function<T>(query : { query: Either<String, Aqb>, ?bindVars : {} }, opts : {}, cb : ArangoCallback<Cursor<T>>) : Void {})
```

Comment:
In fact the wrong thing is the `Aqb` typedef. It should not be String but externs for the aqb library (
someting I made in [ExternKit](https://github.com/ExternKit/extern-js-aqb)) but because I don't want to break things for other user I made it very general.

I know all these possibilities look weird but this is the way [it works](https://github.com/arangodb/arangojs/blob/master/src/database.js#L187).

`query`is handled like this:
- If `query` contains a field named `query`, then `query = query.query`.
- If `query` has a `toAQL` function, then `query = query.toAQL()`.